src/tests/rxcpp_benchmark.cpp

```diff
@@ -310,6 +310,21 @@ TEST_CASE("merge")
     };
 }
 
+TEST_CASE("buffer")
+{
+    BENCHMARK_ADVANCED("buffer")(Catch::Benchmark::Chronometer meter)
+    {
+        auto sub = rxcpp::make_subscriber<std::vector<int>>();
+
+        meter.measure([&]
+            {
+                return rxcpp::observable<>::range(1, 5)
```

Comment:
To have correct benchmark results use same code. rpp uses `just(...)`, so, use `just` in rxcpp too

Reply:
That's fair. Besides, I thought I hit the error that there's no matching function for `just(1,2,3,4,5,6,7)`.

Reply:
as far as i remeber it should use `from`